docs: update Hyundai packages

incognitojam opened the pull request:
- Ioniq Electric 2020+ have SCC as standard
- ~~Changed Ioniq Hybrid 2020-22 to just 2022 as harness can't be confirmed (only one dongle, 2022)~~ check hyundai tech info
- Brochures couldn't be found for the Kona Hybrid 2020 and Kia Niro Hybrid 2021, but it is very likely that these cars have LKAS on all SCC trims too.

Review each model and determine whether LKAS in included in the same package/trim as SCC:
Hyundai:
- [x] Elantra
- [ ] Genesis
  - [ ] 2015 - can't find brochure, will keep LKAS for now
  - [x] 2016
- [x] Ioniq Hybrid
  - [x] 2017-19
  - [x] 2020-22
- [x] Ioniq Plug-in Hybrid 2019
- [x] Ioniq Electric 2019
- [ ] Kona Hybrid 2020 - can't find brochure, assume LKAS is on all SCC trims
- [x] Kona Electric 2022
- [x] Sonata 2018-19

Kia:
- [ ] Niro Hybrid
  - [ ] 2021 - can't find brochure, assume LKAS is on all SCC trims
  - [x] 2022
- [x] Sorento
  - [x] 2018
  - [x] 2019
- [x] Stinger 2018-20

incognitojam committed:
LKAS is part of same package as SCC for Elantra 2017-2019

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2017-elantra.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2018-elantra.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-elantra.pdf

incognitojam committed:
LKAS is standard on all trims for Elantra 2021

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2021-elantra.pdf

incognitojam committed:
LKAS is part of same package as SCC on Genesis 2016

TODO: confirm 2015

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2016-genesis.pdf

incognitojam committed:
LKAS is part of same package as SCC for Ioniq Hybrid 2017-19

Same package 2017-18
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2017-ioniq.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2018-ioniq.pdf
SEL and Limited on 2019
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-ioniq.pdf

incognitojam committed:
LFA is part of same trim as SCC on Ioniq Hybrid 2020-22

TODO: confirm for 2022

SEL and Limited includes both SCC and LFA
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2020-ioniq.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2021-ioniq.pdf

incognitojam committed:
Remove Ioniq Hybrid 2020-21 until harnesses verified

All three model years were added in a single PR - need to lookup
routes and see if there are users with 2020-21 MY vehicles.
https://github.com/commaai/openpilot/pull/22610

incognitojam committed:
LKAS is on same trim as SCC for Ioniq Electric 2019

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-ioniq.pdf

incognitojam committed:
SCC and LKA are on all trims for Ioniq Electric 2020

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2020-ioniq.pdf

incognitojam committed:
LKA and SCC are on same trims for Ioniq Plug-in Hybrid 2019

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-ioniq.pdf

incognitojam committed:
SCC and LFA are on all trims for Ioniq Plug-in Hybrid 2020-21

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2020-ioniq.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2021-ioniq.pdf

incognitojam committed:
LKAS is standard on Kona Electric 2018-21

SCC only on Ultimate trim, LKAS on all

(2018 UK only) https://www1.hyundai.news/fileadmin/uk/press-kits/20180828_kona_electric/Hyundai_Kona_Electric_-_Technical_data_and_specifications.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-kona.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2020-konaelectric.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2021-konaelectric.pdf

incognitojam committed:
LKAS is part of same package as SCC for Sonata 2018-19

Tech Package includes SCC and LKAS

https://cdn.dealereprocess.org/cdn/brochures/hyundai/2018-sonata.pdf
https://cdn.dealereprocess.org/cdn/brochures/hyundai/2019-sonata.pdf

incognitojam committed:
define LKAS package explicitly

incognitojam committed:
remove package where default

sshane commented:
I'd just make sure that whenever LKAS or LDWS is in the same trim/package as SCC/ASCC to remove the LKAS mention as it's not needed

incognitojam committed:
Merge branch 'master' into docs-hyundai

incognitojam committed:
LFA part of same trim as SCC for Seltos 2021

https://cdn.dealereprocess.org/cdn/brochures/kia/2021-seltos.pdf

incognitojam committed:
LKAS is on all trims with SCC for Sorento 2019

https://cdn.dealereprocess.org/cdn/brochures/kia/2019-sorento.pdf

incognitojam committed:
LKAS is on all trims with SCC for Stinger 2018-20

https://cdn.dealereprocess.org/cdn/brochures/kia/2018-stinger.pdf
https://cdn.dealereprocess.org/cdn/brochures/kia/2019-stinger.pdf
https://cdn.dealereprocess.org/cdn/brochures/kia/2020-stinger.pdf

incognitojam committed:
remove min steer speed definition from docs for Optima 2016

incognitojam committed:
LFA is included in same trim as SCC for Niro Hybrid 2022

https://cdn.dealereprocess.org/cdn/brochures/kia/2022-niro.pdf

incognitojam committed:
update docs

incognitojam committed:
remove Ioniq Hybrid 2020-21, add note

incognitojam committed:
update docs

incognitojam committed:
remove LKAS for Kona Hybrid 2020 and Niro Hybrid 2021, likely included in SCC trims

incognitojam committed:
update docs

incognitojam committed:
Revert "remove Ioniq Hybrid 2020-21, add note"

This reverts commit 62096a8db3fb21771a04e20714ecee906a5c0b34.

incognitojam committed:
Revert "Remove Ioniq Hybrid 2020-21 until harnesses verified"

This reverts commit ac773285f76a108c32d8692d12eef18cabcfd023.

incognitojam committed:
update docs

incognitojam committed:
Merge branch 'master' into docs-hyundai
